request.go

```diff
@@ -212,17 +212,21 @@ func VerifyHTTPRequest(
 		util.GetLogger(req.Context()).WithError(err).Print("Error parsing HTTP headers")
 		return nil, util.MessageResponse(400, "Bad Request")
 	}
-	if request.fields.Destination != "" {
-		switch {
-		case isLocalServerName != nil && !isLocalServerName(request.fields.Destination):
-			fallthrough
-		case isLocalServerName == nil && destination != request.fields.Destination:
-			message := fmt.Sprintf("Unrecognised server name %q for Destination", request.fields.Destination)
-			util.GetLogger(req.Context()).Warn(message)
-			return nil, util.MessageResponse(400, message)
+	if destination != "" {
```

Comment:
Why? Does this risk breaking validation?

Reply:
huh. I totally forgot about this.
it seems unnecessary now so I'll just rip it out.